.github/workflows/release-step-1.yml

```
@@ -281,11 +284,16 @@ jobs:
               run: |
                   npx semantic-release --dry-run
 
+            # NPM Publish happens here
             - name: Release
               if: ${{ github.event.inputs.dryRun == 'false'}}
               run: |
                   npx semantic-release
 
+            - name: Get Package Version Number
+              uses: martinbeentjes/npm-get-version-action@v1.3.1
```

```suggestion
              uses: martinbeentjes/npm-get-version-action@3cf273023a0dda27efcd3164bdfb51908dd46a5b #v1.3.1
```

for 3rd party (eg non-github-owned) actions you should use a hardcoded git-sha. otherwise, they could change the tag out from underneath you. i grabbed the 1.3.1 sha from here: https://github.com/martinbeentjes/npm-get-version-action/commits/main/